### DISCUSSION TITLE
Buzz's Issue in translating and exporting audio, srt file

### DISCUSSION THREAD

**Discussion by lt700** [community/community#126138] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi there,
I have just bought Buzz app from Apple App store and tried with a movie. when I select to a movie to translate that movie into Vietnamese, it showed srt in English, and exported audio file not in Vietnamese, it exported into chinese original language.

Could you pls fix up the issue? the movie must be translated in Vietnamese when I select Vietnamese language, and export to Vietnamese audio, and srt into Vietnamese. 

I hope you fix up this issue ASAP and update for me, if you don't fix it, your app seems to be useless for me. 

I have also sent you email, but I haven't received any your response. 

Many thanks,

**Comment by queenofcorgis** [comment]
Hi @lt700 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Buzz. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
